src/modules/Builder/pages/BuilderActivityFlow/BuilderActivityFlow.tsx

```diff
@@ -1,28 +1,31 @@
 import { useTranslation } from 'react-i18next';
-import { useNavigate } from 'react-router-dom';
+import { useNavigate, useParams, generatePath } from 'react-router-dom';
 
 import { StyledDirectoryUpButton, StyledBody } from 'shared/styles/styledComponents';
 import { LinkedTabs, Svg } from 'shared/components';
 import { useBreadcrumbs } from 'shared/hooks';
 import { page } from 'resources';
 
-import { newActivityFlowTabs } from './BuilderActivityFlow.const';
+import { getActivityFlowTabs } from './BuilderActivityFlow.utils';
 
 export const BuilderActivityFlow = () => {
+  const { appletId, activityFlowId } = useParams();
   const { t } = useTranslation();
   const navigate = useNavigate();
   useBreadcrumbs();
 
+  const appletActivityFlowUrl = generatePath(page.builderAppletActivityFlow, { appletId });
```

Comment:
please check the same places with path generation when we generate a child of an applet
```suggestion
  const appletActivityFlowUrl = generatePath(page.builderAppletActivityFlow, { appletId | Path.NewApplet });
```